README.md

```diff
@@ -554,8 +554,14 @@ also be read using `ssb.db.onMsgAdded.value`.
 
 ### getStatus
 
-Gets the current db status, same functionality as
-[db.status](https://github.com/ssbc/ssb-db#dbstatus) in ssb-db.
+Gets the current db status, same functionality (observable) as
```

Comment:
I have only used it like
```
ssb.db2.getStartus()
```

does observeable mean you can go:

```js
const stop = ssb.db2.getStatus(status => {
    console.log(status)
})

stop()
```

Reply:
yes :)

Reply:
I think saying you can use it both ways would be good

Example usages:

```js
const status = ssb.db2.getStatus()


const unsbscribe = ssb.db2.getStatus(status => console.log(status.progress)) 

// later
unsubscribe() // unsubscribes the listener from status updates
```

Reply:
Well you can always call .value on an obz (example 1)